doc/02-installation.md

```diff
@@ -564,7 +564,7 @@ The `icingadb-redis` package automatically installs the necessary systemd unit f
 Please run the following command to enable and start its service:
 
 ```bash
-systemctl enable --now icingadb-redis
+systemctl enable --now icingadb-redis-server
```

Comment:
The documentation is correct, the packages themselves have to be fixed.

(FWIW: https://git.icinga.com/packages/icingadb-redis/-/merge_requests/4)

Reply:
They've been fixed: https://icinga.com/blog/2024/06/26/new-redis-open-source-server-packages-for-icinga-db/